### PR TITLE
remove remains of label col

### LINKED ISSUE #1176: Update throughout for label-col no longer existing

### PR DESCRIPTION
**Pull request by na3d**
Closes #1176

### CODE REVIEW

File: src/linodes/linode/settings/layouts/EditConfigPage.js [hunk 1]
```diff
@@ -385,9 +385,9 @@ export class EditConfigPage extends Component {
         <h3 className="sub-header">Virtual Machine Mode</h3>
         <div>
           <fieldset className="form-group row">
-            <div className="col-sm-2">
-              <legend>Virtualization mode</legend>
-            </div>
+            <label className="col-sm-2 col-form-label">
```
Comment (eatonphil) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: You dropped legend here. It's not currently supported by col-form-label anyway... Idk if @mparke had plans to address that.

File: src/linodes/linode/settings/layouts/EditConfigPage.js [hunk 2]
```diff
@@ -397,11 +397,9 @@ export class EditConfigPage extends Component {
         <h3 className="sub-header">Block Device Assignment</h3>
         <div>
           <div className="form-group row">
-            <div className="col-sm-2">
-              <label
-                htmlFor="config-kernel"
-              >Kernel</label>
-            </div>
+            <label className="col-sm-2 col-form-label">
```
Comment (eatonphil) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: You dropped an `htmlFor`

Comment (na3d) [inline src/linodes/linode/settings/layouts/EditConfigPage.js]: oh!  Nice catch!

**Comment by eatonphil** [comment]
At this point I'd just drop all changes to EditConfigPage. @abemassry should have sorted that page out.

**Comment by abemassry** [comment]
Just checked, all references to label-col have been removed from the EditConfigPage

**Pull request merged**